CMakeLists.txt

```diff
@@ -197,6 +214,26 @@ install(FILES package.xml DESTINATION share/libnabo)
 # Register the local build in case one doesn't use "make install"
 export(PACKAGE libnabo)
 
+include(GNUInstallDirs)
+
+# 'make install' to the correct locations (provided by GNUInstallDirs).
+install(TARGETS ${LIB_NAME} EXPORT ${PROJECT_NAME}-config
+    ARCHIVE  DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    LIBRARY  DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME  DESTINATION ${CMAKE_INSTALL_BINDIR})  # This is for Windows
+#install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${LIB_NAME})
```

Comment:
Does the above install belong to this library?

Reply:
Yes :-) 
It's to create the library "libnabo::libnabo", to be used by other projects, just with the same name that it would have if included via `find_package(libnabo)`.